crates/pop-common/Cargo.toml

```diff
@@ -19,6 +19,7 @@ serde.workspace = true
 strum.workspace = true
 thiserror.workspace = true
 tokio.workspace = true
+toml_edit.workspace = true
```

Comment:
I dont think we need this anymore with the manifest module added, using cargo_toml dependency.

Reply:
Removed